src/native/corehost/test/nativehost/nativehost.cpp

```diff
@@ -72,7 +72,7 @@ int main(const int argc, const pal::char_t *argv[])
             }
 
             nethost_path = get_directory(nethost_path);
-            nethost_path.append(MAKE_LIBNAME("nethost"));
+            nethost_path.append(LIB_FILE_NAME_X("nethost"));
```

Comment:
```suggestion
            nethost_path.append(LIB_FILE_NAME_X("nethost"));
```
For my education - why is `LIB_FILE_NAME` not sufficient here?

Reply:
All the host components have strings (`pal::char_t`/`pal::string_t`) as `wchar_t`/`std::wstring` for Windows and `char`/`std::string` for non-Windows internally and for exported APIs, so this uses `LIB_FILE_NAME_X` to define it according to the platform.